indexing.sql

```diff
@@ -0,0 +1,51 @@
+-- 33ms
+SELECT DISTINCT make_title
+FROM car_models
+WHERE make_code = 'LAM';
+
+-- 18ms
+SELECT DISTINCT model_title
+FROM car_models
+WHERE make_code = 'NISSAN' and model_code = 'GT-R';
+
+-- 7ms
+SELECT make_code, model_code, model_title, year
+FROM car_models
+WHERE make_code = 'LAM';
+
+-- 233ms
+SELECT *
+FROM car_models
+WHERE year between 2010 and 2015;
+
+-- 55ms
+SELECT *
+FROM car_models
+WHERE year=2010;
+
+CREATE index UX_make_model ON car_models (make_code, model_code);
```

Comment:
This should be two separate indexes. Your index should be based on your `WHERE` clauses. There are no `WHERE` clauses that use both make_code and model_code.